src/libraries/Common/tests/Tests/System/StringTests.cs

```diff
@@ -5312,12 +5312,40 @@ private static void ToLower_Culture(string input, string expected, CultureInfo c
             }
         }
 
+        public static IEnumerable<object[]> ToLower_Invariant_TestData()
+        {
+            yield return new object[] { "", "" };
+            yield return new object[] { "Ab", "ab" };
+            yield return new object[] { "H-/", "h-/" };
+            yield return new object[] { "Hello", "hello" };
+            yield return new object[] { "hElLo", "hello" };
+            yield return new object[] { "AbcdAbc", "abcdabc" };
+            yield return new object[] { "AbcdAbcd", "abcdabcd" };
+            yield return new object[] { "AbcdAbcd/", "abcdabcd/" };
+            yield return new object[] { "AbcdAbcd/-", "abcdabcd/-" };
+            yield return new object[] { "AbcdAbcd/-_", "abcdabcd/-_" };
+            yield return new object[] { "AbcdAbcd-bcdAbc", "abcdabcd-bcdabc" };
+            yield return new object[] { "AbcdAbcd-bcdAbcd", "abcdabcd-bcdabcd" };
+            yield return new object[] { "AbcdAbcd-bcdAbcdA", "abcdabcd-bcdabcda" };
+            yield return new object[] { "AbcdAbcd-bcdAbcdA/", "abcdabcd-bcdabcda/" };
+            // Non-ASCII char:
+            yield return new object[] { "\u436", "\u436" };
```

Comment:
Unlike `\x`, `\u` has a fixed length of four: `$"\\u{(int)someChar:x4}"`

```suggestion
            yield return new object[] { "\u0436", "\u0436" };
```

Reply:
Yep, already fixed, thanks 🙂